### ISSUE TITLE
a11y: Page should start with a level 1 heading

### ISSUE THREAD

**Issue by juliannemarik** [swagger-api/swagger-ui#10480]
### Q&A
 - OS: [e.g. macOS]: macOS
 - Browser: [e.g. chrome, safari]: Chrome
 - Version: [e.g. 22]: 136
 - Method of installation: [e.g. npm, dist assets]: `@nestjs/swagger` via npm
 - Swagger-UI version: [e.g. 3.10.0]: @nestjs/swagger version 11.2.0 which uses Swagger UI version **5.21.0**.
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]: OpenAPI version 3.0

### Content & configuration
Basic/default configuration

### Describe the bug you're encountering
The main heading of the page is not using an `h1` heading:

<img width="1200" src="https://github.com/user-attachments/assets/7430b654-743c-4ffb-a101-e6853081eafe" />

### Expected behavior
`h1` should be used for the main heading of the page to meet accessibility requirements:

<img width="350" src="https://github.com/user-attachments/assets/34fb77de-b230-4db2-a8d8-14d585d872f1" />

**Comment by SkullShoT007** [comment]
Hi, I would like to work on this issue, can you assign this to me.
